### DISCUSSION TITLE
http/protobuf as a default protocol

### DISCUSSION THREAD

**Discussion by kes2464** [open-telemetry/opentelemetry-java-instrumentation#10269] [Q&A] [answered]
Just noticed that from v2.0.0, http/protobuf is the default exporter protocol. 

If I understood it correctly, it is to align with the spec, however the spec also said it is ok if there's a good reason to stay with grpc for individual SDKs.

I came across these discussions and understood why the spec wants http/protobuf to be the default

https://github.com/open-telemetry/opentelemetry-specification/issues/1885
https://github.com/open-telemetry/opentelemetry-specification/pull/1969

but I am just wondering if there was any other reason such as performance, grpc's own issue or anything else that also led to this decision for java SDK/agent specifically?

If it was just purely to align with the spec, I will probably keep using grpc.

Thanks

**Comment by laurit** [comment, marked as answer]
As far as I remember it was mostly to align with the spec. If you so wish you can check the notes from the SIG meeting https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw and watch the recordings of the relevant SIG meetings.
